sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -19,6 +19,29 @@
     secondaryLabel="Previous section"
     class="mx-0"
   >
+    <template #primary-buttons v-if="isLastPage">
+      <span>
+        <v-btn
+          color="primary"
+          v-if="!notDraft && !isFirstPage && !processing"
```

Comment:
When the condition gets complex, we can simplify using a computed property like this. 

![image](https://github.com/user-attachments/assets/467ce4c6-b0f9-4db0-8e48-7577984496db)

![image](https://github.com/user-attachments/assets/967bd61c-6a2c-459f-9e18-33f1388672f2)